packages/web3-eth/src/web3_subscriptions.ts

```diff
@@ -30,9 +30,9 @@ export class LogsSubscription extends Web3Subscription<
 		data: any;
 	},
 	{
-		fromBlock?: number;
-		address?: HexString | HexString[];
-		topics?: (HexString | null)[];
+		readonly fromBlock?: BlockNumberOrTag;
+		readonly address?: Address | Address[];
+		readonly topics?: (Topic | Topic[] | null)[];
```

Comment:
does topics ever return a single `Topic` type or always an `Topic[]` array of Topic